packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md

```diff
@@ -7,7 +7,18 @@ layout: Guides
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-After you have successfully [created and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/) and [downloaded and set up the SDK](/docs/guides/oie-embedded-common-download-setup-app), you can run both the sample Widget app and the sample SDK app.
+This guide covers how you can run both the sample Widget app and the sample SDK app, after you have successfully [created and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/) and [downloaded and set up the SDK](/docs/guides/oie-embedded-common-download-setup-app).
```

Comment:
Weird, in the rendered version this summary appears all in heading font. Again, needs breaking lines?

Reply:
Fixed.